egs/wham/DPRNN/train.py

```diff
@@ -66,31 +66,25 @@ def main(conf):
 
     # Define callbacks
     checkpoint_dir = os.path.join(exp_dir, 'checkpoints/')
-    checkpoint = ModelCheckpoint(checkpoint_dir, monitor='val_loss',
-                                 mode='min', save_top_k=5, verbose=1)
+    checkpoint = ModelCheckpoint(checkpoint_dir, monitor='val_loss', verbose=True,
+                                 mode='min', save_top_k=5)
     early_stopping = False
     if conf['training']['early_stop']:
         early_stopping = EarlyStopping(monitor='val_loss', patience=30,
                                        verbose=1)
 
     # Don't ask GPU if they are not available.
     gpus = -1 if torch.cuda.is_available() else None
-    trainer = pl.Trainer(max_nb_epochs=conf['training']['epochs'],
+    trainer = pl.Trainer(max_epochs=conf['training']['epochs’],
```

Comment:
The quote is broken (`’` instead of `'`)

Reply:
```suggestion
     trainer = pl.Trainer(max_epochs=conf['training']['epochs'], 
```